frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelPropertyCorrelationTable.tsx

```diff
@@ -121,12 +122,16 @@ export function FunnelPropertyCorrelationTable(): JSX.Element | null {
                         CORRELATED PROPERTIES
                     </span>
                     <span className="table-options">
-                        <p className="title">PROPERTIES </p>
-                        <PropertyNamesSelect
-                            value={new Set(propertyNames)}
-                            onChange={(selectedProperties: string[]) => setPropertyNames(selectedProperties)}
-                            allProperties={inversePropertyNames(excludedPropertyNames || [])}
-                        />
+                        {allProperties.length > 0 && (
+                            <>
+                                <p className="title">PROPERTIES </p>
+                                <PropertyNamesSelect
+                                    value={new Set(propertyNames)}
```

Comment:
Not blocking - we're constructing a new set here, I wonder if it's causing uneeded rerenders?

Reply:
I'll have a look outside this PR!